### ISSUE TITLE
QuickReplant not working with Farmer's Delight Crops

### ISSUE THREAD

**Issue by Miv2nir** [svenhjol/Charm#820]
Right clicking with a hoe does work with vanilla crops but unfortunately does absolutely nothing on Farmer's Delight Crops. It would be nice to have a support for it.

**Comment by brotazoa** [comment]
Same issue on 1.19.2 fabric. Adding entries to the config does nothing.

**Comment by DesertDraggon** [comment]
Agreed, same issue in 1.19.2 fabric. Along with other mods like Better End and Better Nether. Adding crops VIA config doesn't appear to work properly. Animation of using hoe on crops does activate on BE and BN, but no actual harvesting happens. 

Even attempted to add minecraft:cocoa VIA config, but same result: no harvesting or replanting.
